apimodels/perf.go

```diff
@@ -0,0 +1,52 @@
+package apimodels
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/evergreen-ci/timber"
+	perf "github.com/evergreen-ci/timber/perf"
```

Comment:
This doesn't have to be explicitly labeled as "perf" since the package is already called perf.

Reply:
Agreed. I got a linting error I resolved by adding this for some reason a while back, but this is no longer the case...